cypress/plugins/index.js

```diff
@@ -22,9 +23,17 @@ module.exports = (on, config) => {
   on("file:preprocessor", webpack({
     webpackOptions: {
       resolve: {
-        extensions: [".ts", ".js"]
+        extensions: [".ts", ".js"],
+        alias: {
+          process: 'process/browser',
```

Comment:
Just picture to show tests are passing
![image](https://user-images.githubusercontent.com/30693990/153625815-e6084f80-8ae2-40cd-9dda-27237be519a8.png)